prometheus-exporters/vrops-exporter/alerts/host.alerts

```diff
@@ -356,3 +356,17 @@ groups:
     annotations:
       description: "HA on Host `{{ $labels.hostsystem }}` has detected a network-partitioned host. ({{ $labels.vcenter }})."
       summary: "HA on Host `{{ $labels.hostsystem }}` has detected a network-partitioned host. ({{ $labels.vcenter }})."
+      
+  - alert: APodHostRunsMultipleVCenters
+    expr: count(vrops_virtualmachine_runtime_connectionstate{vccluster=~".*-management", virtualmachine=~"vc-.*"}) by(hostsystem) > 1 
```

Comment:
this will also include `vrops-vc` VMs, ist this wanted?

Reply:
Why? The regex is "vc-.*" shouldn't inlude the vrops VMs, right?